azkaban-webserver/src/web/js/azkaban/view/schedule-panel.js

```diff
@@ -90,4 +74,146 @@ $(function() {
   schedulePanelView =  new azkaban.SchedulePanelView({
     el: $('#schedule-modal')
   });
+
+
+  var TimeZoneOffset = new Date().toString().match(/([-\+][0-9]+)\s/)[1];
+  $('#timeZoneOffset').html(TimeZoneOffset);
+
+  updateOutput();
+  $("#clearCron").click(function () {
+    $('#cron-output').val("* * * * ?");
+    resetLabelColor();
+    $("#minute_input").val("*");
+    $("#hour_input").val("*");
+    $("#dom_input").val("*");
+    $("#month_input").val("*");
+    $("#dow_input").val("?");
+    $(cron_translate_id).text("")
+    $(cron_translate_warning_id).text("")
+    $('#nextRecurId').html("");
+
+    while ($("#instructions tbody tr:last").index() >= 4) {
+      $("#instructions tbody tr:last").remove();
+    }
+  });
+
+  $("#minute_input").click(function () {
```

Comment:
Add a comment: this is to update the instruction panel based on which field is selected?